src/components/MainPage/Importers/Importer.vue

```diff
@@ -1,52 +1,70 @@
 <template>
-  <div v-if="decryptedImporter">
-    <div>
-      <el-checkbox v-model="showBrowser">
-        Show browser
-      </el-checkbox>
-    </div>
-    <div
-      v-for="(value, loginField) in decryptedImporter.loginFields"
-      :key="loginField"
-    >
-      {{ loginField }}:
-      {{ loginField != "password" ? value : "*".repeat(value.length) }}
-    </div>
-    <el-button
-      type="primary"
-      :loading="importing"
-      @click="scrape()"
-    >
-      Import
-    </el-button>
-    <el-button
-      type="danger"
-      icon="el-icon-delete"
-      :disabled="importing"
-      @click="promptDelete()"
-    >
-      Delete
-    </el-button>
-    <el-progress
-      v-show="importing"
-      :percentage="percentage"
-      :status="status"
-      class="progress-bar"
+  <div>
+    <v-container v-if="decryptedImporter">
```

Comment:
And this container should be designed by the `v-list` components. Not with `div`s that not creating a space between items.